DCU version compile error

### Describe the bug

With the latest commit `c61a66eaabf0` (8-24), the dcu ABACUS is compiled failed with error:
```
clang-15: warning: argument unused during compilation: '-amdgpu-function-calls=false' [-Wunused-command-line-argument]
[ 90%] Linking CXX executable abacus_pw
/opt/rh/devtoolset-7/root/usr/lib/gcc/x86_64-redhat-linux/7/../../../../bin/ld: Dwarf Error: found dwarf version '5', this reader only handles version 2, 3 and 4 information.
source/module_hamilt_pw/hamilt_pwdft/CMakeFiles/hamilt_pwdft.dir/forces_cc.cpp.o: In function `Forces<double, base_device::DEVICE_GPU>::cal_force_cc(ModuleBase::matrix&, ModulePW::PW_Basis*, Charge const*, UnitCell&)':
forces_cc.cpp:(.text._ZN6ForcesIdN11base_device10DEVICE_GPUEE12cal_force_ccERN10ModuleBase6matrixEPN8ModulePW8PW_BasisEPK6ChargeR8UnitCell[_ZN6ForcesIdN11base_device10DEVICE_GPUEE12cal_force_ccERN10ModuleBase6matrixEPN8ModulePW8PW_BasisEPK6ChargeR8UnitCell]+0xa4a): undefined reference to `hamilt::cal_force_npw_op<double, base_device::DEVICE_GPU>::operator()(std::complex<double> const*, double const*, double const*, double const*, double const*, double*, double, double, double, int, double, double)'
forces_cc.cpp:(.text._ZN6ForcesIdN11base_device10DEVICE_GPUEE12cal_force_ccERN10ModuleBase6matrixEPN8ModulePW8PW_BasisEPK6ChargeR8UnitCell[_ZN6ForcesIdN11base_device10DEVICE_GPUEE12cal_force_ccERN10ModuleBase6matrixEPN8ModulePW8PW_BasisEPK6ChargeR8UnitCell]+0xaff): undefined reference to `hamilt::cal_force_npw_op<double, base_device::DEVICE_GPU>::operator()(std::complex<double> const*, double const*, double const*, double const*, double const*, double*, double, double, double, int, double, double)'
clang-15: error: linker command failed with exit code 1 (use -v to see invocation)
```

### Expected behavior

_No response_

### To Reproduce

_No response_

### Environment

_No response_

### Additional Context

_No response_

### Task list for Issue attackers (only for developers)

- [ ] Verify the issue is not a duplicate.
- [ ] Describe the bug.
- [ ] Steps to reproduce.
- [ ] Expected behavior.
- [ ] Error message.
- [ ] Environment details.
- [ ] Additional context.
- [ ] Assign a priority level (low, medium, high, urgent).
- [ ] Assign the issue to a team member.
- [ ] Label the issue with relevant tags.
- [ ] Identify possible related issues.
- [ ] Create a unit test or automated test to reproduce the bug (if applicable).
- [ ] Fix the bug.
- [ ] Test the fix.
- [ ] Update documentation (if necessary).
- [ ] Close the issue and inform the reporter (if applicable).

--- a/source/module_hamilt_pw/hamilt_pwdft/kernels/rocm/stress_op.hip.cu
+++ b/source/module_hamilt_pw/hamilt_pwdft/kernels/rocm/stress_op.hip.cu
@@ -743,4 +743,7 @@ template struct cal_dbecp_noevc_nl_op<double, base_device::DEVICE_GPU>;
 template struct cal_stress_nl_op<float, base_device::DEVICE_GPU>;
 template struct cal_stress_nl_op<double, base_device::DEVICE_GPU>;
 
+template struct cal_force_npw_op<double, base_device::DEVICE_GPU>;
+template struct cal_force_npw_op<float, base_device::DEVICE_GPU>;
+
 }  // namespace hamilt

